packages/mdc-theme/README.md

```diff
@@ -36,8 +38,6 @@ In addition, it's important to consider the style of text:
 to establish a visual identity and color many parts of your application. The latter refers to the style of text
 that is most prominent (low opacity, high contrast), and used to display most content.
 
-The text contrast colors are automatically calculated at the Sass level and exposed as part of this module.
```

Comment:
We have several issue regarding this, worth to provide a paragraph explaining this.
#608 #755